tests/Unit/Http/GraphQLHandlerTest.php

```
@@ -0,0 +1,257 @@
+            static fn (Asset $asset): bool => (string) $asset->getId() === (string) $assetId,
+        );
+    }
+
+    public function findByUploadId(UploadId $uploadId): ?Asset
+    {
+        return $this->findSavedAsset(
+            static fn (Asset $asset): bool => (string) $asset->getUploadId() === (string) $uploadId,
+        );
+    }
+
+    public function searchByFileName(AccountId $accountId, string $query): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$accountId'. (undefined)

(UnusedFormalParameter)